package.json

```diff
@@ -2,18 +2,22 @@
   "name": "@ngrx/store",
   "version": "2.1.2",
   "description": "RxJS powered Redux for Angular2 apps",
-  "main": "./index.js",
+  "main": "./bundles/store.umd.js",
+  "module": "./index.js",
   "scripts": {
-    "clean": "rimraf node_modules typings dist tmp npm-debug.log && npm cache clean",
-    "clean:test": "rimraf tmp",
-    "clean:dist": "rimraf dist",
-    "build:dist": "npm run clean:dist && ngc && cp package.json dist/package.json",
-    "lint": "tslint src/**.ts",
-    "test": "tsc -p spec && jasmine && npm run clean:test",
-    "test:ngc": "ngc -p spec/tsconfig.ngc.json",
-    "typings": "typings install",
-    "prebuild:dist": "npm run typings",
-    "prepublish:dist": "npm run build:dist"
+    "karma": "karma start --single-run",
+    "test:unit": "npm run karma",
+    "test:ngc": "ngc -p ./spec/ngc/tsconfig.ngc.json",
+    "test": "npm run test:unit && npm run test:ngc",
+    "clean:pre": "rimraf release",
+    "clean:post": "rimraf src/**/*.ngfactory.ts",
```

Comment:
This might not matter if the assumption is that every build will occur for every collaborator on a UNIX flavored machine. If you want reliably good results across platforms though, it is unfortunately necessary to quote parameters that appear in the scripts section and which contain wildcards, to ensure that the wild karting occurs by the target tool and not by the command processors which vary between platforms. So if that matters, you would write something ugly like:

```
 "clean:post": "rimraf \"src/**/*.ngfactory.ts\"",
```

This is of course, so offensive to the eye that I will happily run away with it and never mention it here again if cross-platform does not matter...